helm/publisher/values.yaml

```diff
@@ -1,4 +1,9 @@
 images:
   articleStore:
     repository: liberoadmin/article-store
-  pullPolicy: IfNotPresent
+
+postgresql:
+  nameOverride: article-store-postgresql
+  persistence:
+    enabled: false
+  postgresqlPassword: pAsSw0rD
```

Comment:
Default chart behaviour is to update the secret on every release, but the change is not propagated. So, future app pods get the new version, but the Postgres pod _doesn't_ (unless it happens to be recreated on the same release _and_ has no persistence).

tl;dr: set secret values manually when using Helm

Refs https://github.com/helm/charts/issues/5167